opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -3189,7 +3189,7 @@ interface Container {
     types: [String]
     viaTypes: [String]
   ): ContainerConnection
-  contentsFromTemplate(
+  filesFromTemplate(
```

Comment:
I don't understand why we have all these arguments in this query (first, prefixMimeType, after, orderBy, orderMode, search, filters), if we only use first on the frontend ? and in the resolver we only extract first and prefixMimeType, and we don't use the other arguments.
I know it's not directly related to this PR which only renames this API, but I just noticed it.

Reply:
filesFromTemplate are exactly the same nature has importFiles, so for consistancy, the same arguments have been made available